README.md

```diff
@@ -14,7 +14,7 @@ Vulkan also supports multiple global contexts (instances, in Vulkan terminology)
 The ICD loader is a library that is placed between a Vulkan application and any number of Vulkan drivers, in order to support multiple drivers and the instance-level functionality that works across these drivers.
```

Comment:
Do we need to update anything in this section? Maybe put a "### Vulkan" header above any Vulkan specific paragraphs?

Reply:
I've tried to explain the Vulkan vs Vulkan SC support better in the latest version